### PR TITLE
Bump revisions for fastrpc, qrtr, rmtfs tqftpserv and skales recipes

### PR DESCRIPTION
**Pull request by quic-vkraleti**
Bumped SRCREV for fastrpc, qrtr, rmtfs tqftpserv and skales recipes to pick latest commits from respective projects and build.

### CODE REVIEW

File: recipes-support/fastrpc/fastrpc_git.bb [hunk 1]
```diff
@@ -28,7 +29,7 @@ SYSTEMD_PACKAGES = "${PN} ${PN}-systemd"
 
 SYSTEMD_SERVICE:${PN} = "usr-lib-rfsa.service"
 
-SYSTEMD_SERVICE:${PN}-systemd = "adsprpcd.service cdsprpcd.service"
+SYSTEMD_SERVICE:${PN}-systemd = "adsprpcd.service cdsprpcd.service sdsprpcd.service"
 SYSTEMD_AUTO_ENABLE:${PN}-systemd = "disable"
```
Comment (lumag) [inline recipes-support/fastrpc/fastrpc_git.bb:33]:
```
Parsing recipes...ERROR: ../meta-qcom/recipes-support/fastrpc/fastrpc_git.bb: Unable to get checksum for fastrpc SRC_URI entry sdsprpcd.service: file could not be found
```

**Review by lumag — CHANGES_REQUESTED** [review]

**Comment by lumag** [comment]
Also, could you please take a look at the surrounding commits and follow the existing  subject style.

File: recipes-support/fastrpc/fastrpc_git.bb (hunk 1 above)
Comment (quic-vkraleti) [reply recipes-support/fastrpc/fastrpc_git.bb:33]: Adding sdsprpc.service under files meta-qcom/recipes-support/fastrpc/fastrpc folder

File: recipes-support/qrtr/qrtr_git.bb [hunk 2]
```diff
@@ -7,14 +7,14 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=15329706fbfcb5fc5edcc1bc7c139da5"
...
-SRCREV = "9dc7a88548c27983e06465d3fbba2ba27d4bc050"
+SRCREV = "d0d471c96e7d112fac6f48bd11f9e8ce209c04d2"
 SRC_URI = "git://github.com/andersson/${BPN}.git;branch=master;protocol=https"
 
 PV = "0.3+${SRCPV}"
 
 S = "${WORKDIR}/git"
 
-EXTRA_OEMAKE = "prefix=${prefix} bindir=${bindir} libdir=${libdir} includedir=${includedir} LDFLAGS='${LDFLAGS}' CFLAGS='${CFLAGS}'"
+EXTRA_OEMAKE = "prefix=${prefix} bindir=${bindir} libdir=${libdir} includedir=${includedir} LDFLAGS='${LDFLAGS}'"
```
[the remark this answers is not in the document]
Comment (quic-vkraleti) [reply recipes-support/qrtr/qrtr_git.bb]: Removed both.

**Comment by quic-vkraleti** [comment]
I am not able to cleanly update commits. I am taking down this request and 'll submit another one.

**Comment by lumag** [comment]
@quic-vkraleti No. Please don't do that. Reopening.

**Comment by lumag** [comment]
Please find a good Git manual and study how to work with commits. Opening new PR for the same featureset means killing previous review and conversations, thus it is frowned upon.

**Comment by lumag** [comment]
No need to merge master. It's done automatically during the merge. Please remove the merge commit.

**Comment by lumag** [comment]
Please fix commit sujects to follow 'name: description' instead of 'recipes-foo: bar do something'.

**Review by lumag — APPROVED** [review]

**Pull request merged**